doc/userguide/src/ExecutingTestCases/OutputFiles.rst

```diff
@@ -552,6 +552,32 @@ deeply nested keyword structures.
 
 .. note:: `TAG:<pattern>` mode was added in Robot Framework 2.9.
 
+Automatically expanded keywords in log file
+-------------------------------------------
+
+Keywords that have passed are closed in the log file by default. Thus information 
+they contain is not visible unless you expand them. If certain keywords have
+important information that should be visible when the log file is opened, you can 
+use the :option:`--expandkeywords` option to set keywords automatically expanded
+in log file similar to failed keywords.
+
+The :option:`--expandkeywords` option supports the following modes:
+
+`NAME:<pattern>`
+   Expand keywords matching the given pattern. Pattern matching rules are
+   same as when `removing keywords`_ using `NAME:<pattern>` mode.
+
+`TAG:<pattern>`
+   Expand keywords with tags matching the given pattern. Pattern matching
+   rules are same as when `removing keywords`_ using `TAG:<pattern>` mode.
+
+Examples::
+
+   rebot --expandkeywords name:CapturePageScreenshot output.xml
+   rebot --expandkeywords tag:ShowInLog output.xml
```

Comment:
This works both with Robot and Rebot, right? Changing the first example to `robot ... tests.robot` would probably be a good idea. Not sure should that fact be explicitly mentioned as well, probably not.

Reply:
right. robot+rebot. it's a common option, thus we can set these settings via pabot